packages/@glimmer/util/lib/weak-set.ts

```diff
@@ -0,0 +1,18 @@
+export default (typeof WeakSet === 'function'
```

Comment:
We should make sure that Ember drops its own `WeakSet` polyfill over in the integration PR.